injector/src/test/resources/simple.java

```diff
@@ -1,5 +1,4 @@
 package injector;
-
```

Comment:
why this change?

Reply:
Left over from another change, thank you for noticing this. We used to add an extra new line for each import and this is fixed in previous PR. Brought it back. e48eb1a283e8df35293e92bc19f7e19d1b5fc038